tests/gccgo/task.yaml

```diff
@@ -0,0 +1,14 @@
+summary: Check that snapd builds with gccgo
+prepare: |
+    echo Installing gccgo-6 and pretending it is the default go
+    sudo apt install -y gccgo-6
+    sudo ln -s /usr/bin/go-6 /usr/local/bin/go
+restore: |
+    rm -f /usr/local/bin/go
+    sudo apt-get autoremove -y gccgo-6
+execute: |
+    echo Ensure we really build with gccgo
+    go version|grep gccgo
+    echo Build the deb with gccgo and run the tests as part of the build
+    sudo -i -u test /bin/sh -c "cd /gopath/src/github.com/snapcore/snapd && dpkg-buildpackage -Zgzip"
```

Comment:
Thanks! I added the `-tc` flag now. The `apt autoremove` should remove all dependencies that got installed earlier as part of the go-6 install. I will double check in the spread logs to be sure that its really doing that.